src/Datastore.cpp

```diff
@@ -76,7 +76,7 @@ void DatastoreClass::loop()
         if (inv->isReachable()) {
             isReachable++;
         } else {
-            if (inv->getEnablePolling()) {
+            if (cfg->Poll_Enable) {
```

Comment:
Why is this necessary here but not elsewhere above and below?

Reply:
The original logic was to set all available if they are suposed to be available. That means if you don't fetch inverter values at night, all other inverters are still valid because you intentional don't pull this specific inverter. 
Example: You have two inverters, one at a solar panel, one at a battery (which can run the whole night). If you don't fetch the first one at night, the ac_total should be still valid because the second inverter works as expected (as so does the first one as it is suspected to be offline)